src/main/java/org/epamTask/Drink.java

```diff
@@ -0,0 +1,51 @@
+package org.epamTask;
+
+public class Drink {
+    private String name;
+    private double price;
```

Comment:
Try to use wrapper:
int -> Integer
double -> Double
They have more calculation abilities